### ISSUE TITLE
--bowtie_indices fails

### ISSUE THREAD

**Issue by bounlu** [nf-core/smrnaseq#159]
### Description of the bug

Thanks for releasing the new version `v2.0.0` on the `master`.

I just tested it however specifying iGenomes index for Bowtie failed as below:

```
Execution cancelled -- Finishing pending tasks before exit
Error executing process > 'NFCORE_SMRNASEQ:SMRNASEQ:MIRDEEP2:MIRDEEP2_MAPPER (ST_Qr3)'

Caused by:
  Essential container in task exited

Command executed:

  mapper.pl \
  ST_Qr3_trimmed.fq \
      -e \
      -h \
      -i \
      -j \
      -m \
      -p genome \
      -s ST_Qr3_collapsed.fa \
      -t ST_Qr3_reads_vs_refdb.arf \
      -o 4
  
  cat <<-END_VERSIONS > versions.yml
  "NFCORE_SMRNASEQ:SMRNASEQ:MIRDEEP2:MIRDEEP2_MAPPER":
      mapper: $(echo "2.0.1")
  END_VERSIONS

Command exit status:
  255

Command output:
  (empty)

Command error:
  nxf-scratch-dir ip-172-31-46-27.ap-southeast-1.compute.internal:/tmp/nxf.XXXXPnQQvm
  Could not locate a Bowtie index corresponding to basename "genome"
  Please make sure you used bowtie version 1 to build the index.
  Usual index files have suffix .ebwt

Work dir:
  s3://omeran/nextflow/smrnaseq3/bucket/21/f886db8d65a220243bfaea101f0907

Tip: view the complete command output by changing to the process work dir and entering the command `cat .command.out`

```

### Command used and terminal output

```console
nextflow run nf-core/smrnaseq \
-profile docker \
--genome GRCh38 \
--mirtrace_species "hsa" \
--fasta "s3://ngi-igenomes/igenomes/Homo_sapiens/NCBI/GRCh38/Sequence/WholeGenomeFasta/genome.fa" \
--bowtie_indices "s3://ngi-igenomes/igenomes/Homo_sapiens/NCBI/GRCh38/Sequence/BowtieIndex/" \
--mature 'https://www.mirbase.org/ftp/CURRENT/mature.fa.gz' \
--hairpin 'https://www.mirbase.org/ftp/CURRENT/hairpin.fa.gz' \
--input 'samplesheet_smrnaseq.csv' \
--outdir 's3://omeran/nextflow/smrnaseq3/results/' \
-bucket-dir 's3://omeran/nextflow/smrnaseq3/bucket/' \
-work-dir 's3://omeran/nextflow/smrnaseq3/work/' \
-c 'custom.config' \
-r master
```
```


### Relevant files

_No response_

### System information

```
nextflow info
  Version: 22.04.3 build 5703
  Created: 18-05-2022 19:22 UTC (19-05-2022 03:22 SGST)
  System: Mac OS X 11.4
  Runtime: Groovy 3.0.10 on OpenJDK 64-Bit Server VM 11.0.11+9
  Encoding: UTF-8 (UTF-8)
```

**Comment by bounlu** [comment]
The issue arises from this [line](https://github.com/nf-core/smrnaseq/blob/d1299b132a910d8adb38a5445d0c223051e14264/subworkflows/local/genome_quant.nf#L24). Hope to get a fix on this.

**Comment by bounlu** [comment]
The error is indeed due to failure in `mapper.pl` script in `mirdeep2` [here](https://github.com/rajewsky-lab/mirdeep2/blob/c6440e298795579ad62351bba7aff9cc43c50c68/src/mapper.pl#L413).

**Comment by apeltzer** [comment]
Yeah looks like a small bug to fix. Will give this a try now.

**Comment by apeltzer** [comment]
@JoseEspinosa - just had a look and cannot really say whats the error with that line. Any idea?

**Comment by JoseEspinosa** [comment]
From what I see in the error message, I'd say that the problem is not in the line reported above but is that the `genome.*.ebwt` files are not correctly staged in the `work` directory, but it is weird. I will need to run with the same reference and see if the error can be reproduced. @bounlu could you list the content of the work directory and post it here? (`aws s3 ls s3://omeran/nextflow/smrnaseq3/bucket/21/f886db8d65a220243bfaea101f0907`), also maybe you could double-check that they are exactly the same as the ones in `s3://ngi-igenomes/igenomes/Homo_sapiens/NCBI/GRCh38/Sequence/BowtieIndex/`:
```
$ aws s3 ls s3://ngi-igenomes/igenomes/Homo_sapiens/NCBI/GRCh38/Sequence/BowtieIndex/
2017-04-13 13:22:26  842769182 genome.1.ebwt
2017-04-13 13:22:51  366859564 genome.2.ebwt
2017-04-13 13:23:07      10880 genome.3.ebwt
2017-04-13 13:23:08  733719113 genome.4.ebwt
2017-04-13 13:23:40 3144230986 genome.fa
2017-04-13 13:24:14  842769182 genome.rev.1.ebwt
2017-04-13 13:24:42  366859564 genome.rev.2.ebwt
```

**Comment by JoseEspinosa** [comment]
We fixed it in #165, you can try to give it a try in the dev branch @bounlu otherwise, `2.1.0` is in preparation stay tuned. I will close the issue now but feel free to come back to us if you find any other issue.

**Comment by bounlu** [comment]
Thanks a lot for the fix! I will be testing it soon.
